README.md

```diff
@@ -16,184 +16,74 @@
+- `./setup.sh` - To run Polaris as a mini-deployment locally. This will create two pods that bind themselves to port `8181`.
+- `kubectl get pods` - To check the status of the pods.
+- `kubectl get deployment` - To check the status of the deployment.
+- `kubectl describe deployment polaris-deployment` - To troubleshoot if things aren't working as expected.
+
+Polaris is organized into the following modules:
 - `polaris-core` - The main Polaris entity definitions and core business logic
 - `polaris-server` - The Polaris REST API server
 - `polaris-eclipselink` - The Eclipselink implementation of the MetaStoreManager interface
 
-Build the binary (first time may require installing new JDK version). This build will run IntegrationTests by default.
-Make sure docker is running, as the integration tests require a running docker daemon.
-
-```
-./gradlew build
-```
-
-To skip tests.
-
-```
-./gradlew assemble
-```
-
-Run the Polaris server locally on localhost:8181
-
-```
-./gradlew runApp
-```
-
-The server will start using the in-memory mode, and it will print its auto-generated credentials to STDOUT in a message like the following:
-
-```text
-realm: default-realm root principal credentials: <id>:<secret>
-```
-
-These credentials can be used as "Client ID" and "Client Secret" in OAuth2 requests (e.g. the `curl` command below).
-
-While the Polaris server is running, run regression tests, or end-to-end tests in another terminal
-
-```
-./regtests/run.sh
-```
-
-Docker Instructions
--------------------
-
-Build the image:
-
-```
-docker build -t localhost:5001/polaris:latest .
-```
-
-Run it in a standalone mode. This runs a single container that binds the container's port `8181` to localhosts `8181`:
-
-```
-docker run -p 8181:8181 localhost:5001/polaris:latest
-```
-
-# Running the tests
-
-## Unit and Integration tests
-
-Unit and integration tests are run using gradle. To run all tests, use the following command:
-
-```bash
-./gradlew test
-```
-
-## Regression tests
-
-Regression tests, or functional tests, are stored in the `regtests` directory. They can be executed in a docker
-environment by using the `docker-compose.yml` file in the project root.
-
-```bash
-docker compose up --build --exit-code-from regtest
-```
-
-They can also be executed outside of docker by following the setup instructions in
-the [README](regtests/README.md)
-
-# Kubernetes Instructions
------------------------
-
-You can run Polaris as a mini-deployment locally. This will create two pods that bind themselves to port `8181`:
-
-```
-./setup.sh
-```
-
-You can check the pod and deployment status like so:
-
-```
-kubectl get pods
-kubectl get deployment
-```
-
-If things aren't working as expected you can troubleshoot like so:
-
-```
-kubectl describe deployment polaris-deployment
-```
-
-## Creating a Catalog manually
-
-Before connecting with Spark, you'll need to create a catalog. To create a catalog, generate a token for the root
-principal:
-
+Building docs
```

Comment:
To simplify the README more, we can move the doc building part to a more detailed doc. WDYT?

Reply:
From my POV it is good to have all essential building tips in the top-level README. I suppose contributors will be changing docs often.